packages/react-components/react-table/stories/Table/MultipleSelect.stories.tsx

```diff
@@ -184,3 +184,15 @@ export const MultipleSelect = () => {
     </Table>
   );
 };
+
```

Comment:
Please double check keyboard navigation - I can select the row checkbox by both Enter and Space but Space scrolls.
For the uber checkbox for all lines I can only select it using Space (without scroll) cannot select it using Enter.

Reply:
Good point, I've made it behave consistently to uber checkbox and added `preventDefault` to stop the scrolling